### PR TITLE
chore(): update readme and add lingui/macro as peer dep

### PR DESCRIPTION
**Pull request by thekip**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 <div align="center">
 
-A Rust versions of [LinguiJS Macro](https://lingui.js.org/ref/macro.html) [<img src="https://img.shields.io/badge/beta-yellow"/>](https://github.com/lingui/swc-plugin)
+A Rust versions of [LinguiJS Macro](https://lingui.dev/ref/macro.html) [<img src="https://img.shields.io/badge/beta-yellow"/>](https://github.com/lingui/swc-plugin)
```
Comment (andrii-bodnar) [inline README.md:5]:
```suggestion
A Rust versions of [LinguiJS Macro](https://lingui.dev/ref/macro) [<img src="https://img.shields.io/badge/beta-yellow"/>](https://github.com/lingui/swc-plugin)
```

File: README.md [hunk 2]
```diff
@@ -23,10 +39,10 @@ https://swc.rs/docs/configuration/swcrc
   "jsc": {
     "experimental": {
       "plugins": ["@lingui/swc-plugin", {
-        
+
         // Optional
         // Unlike the JS version this option must be passed as object only.
-        // Docs https://lingui.js.org/ref/conf.html#std-config-runtimeConfigModule
+        // Docs https://lingui.dev/ref/conf.html#runtimeconfigmodule
```
Comment (andrii-bodnar) [inline README.md:45]:
```suggestion
        // Docs https://lingui.dev/ref/conf#runtimeconfigmodule
```

File: package.json [hunk 3]
```diff
@@ -1,30 +1,33 @@
...
-    "main": "target/wasm32-wasi/release/lingui_macro_plugin.wasm",
-    "exports": {
-        ".": "./target/wasm32-wasi/release/lingui_macro_plugin.wasm"
-    },
-    "scripts": {
-        "prepublishOnly": "cargo build-wasi --release"
-    },
-    "files": [],
-    "packageManager": "yarn@3.3.1"
+  "name": "@lingui/swc-plugin",
```
Comment (andrii-bodnar) [inline package.json:2]: Please also add the `repository` and `bugs` fields to let NPM display these links here https://www.npmjs.com/package/@lingui/swc-plugin

**Comment by thekip** [comment]
Done

**Review by andrii-bodnar — APPROVED** [review]

**Pull request merged**